omop_examples/example_demographic_dashboard.ipynb

```diff
@@ -0,0 +1,263 @@
+   "source": [
+    "gender_concept_query = \"\"\"\n",
+    "SELECT concept_id as gender_concept_id, concept_name as gender FROM `concept` where domain_id = \"Gender\"\n",
+    "\"\"\"\n",
+    "gender_concept_df = client.query(gender_concept_query).result().to_dataframe()\n",
+    "gender_concept_df.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3658d3f9-ab44-458f-948e-58e175a338fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "people_query = \"\"\"\n",
+    "SELECT \n",
+    "  person_id,\n",
+    "  race_concept_id,\n",
+    "  gender_concept_id,\n",
+    "  year_of_birth,\n",
+    "  month_of_birth\n",
+    "FROM \n",
+    "  `person`\n",
+    "WHERE RAND() < 10000/2326856\n",
+    "\"\"\"\n",
+    "\n",
+    "cms_syn_df = client.query(people_query).result().to_dataframe()\n",
+    "cms_syn_df.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b48a7001-252f-4571-a68f-1ad1385327be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "merged_df = pd.merge(cms_syn_df, race_concept_df, on=\"race_concept_id\")\n",
+    "merged_df = pd.merge(merged_df, gender_concept_df, on=\"gender_concept_id\")\n",
+    "merged_df.head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21f81ad6-f877-4097-a727-dcd480abba43",
+   "metadata": {},
+   "source": [
+    "# Build example interactive dashboard"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40d49097-182f-4f09-8ac6-698b0d0cf1ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_histogram(bins = 10, hue = 'race', palette = 'Blues', x_range_1 = (1900,2000)): \n",
+    "    plt.figure(dpi = 120)\n",
+    "    sns.histplot(data = merged_df, \n",
+    "                        x = 'year_of_birth',\n",
+    "                        palette=palette, \n",
+    "                        bins = bins, \n",
+    "                        hue = hue,\n",
+    "                       )\n",
+    "    plt.xlim(x_range_1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "90beea2d-99c8-4832-b988-25508db5d3d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = interact(\n",
+    "    plot_histogram,\n",
+    "    palette = widgets.Dropdown(\n",
+    "        options = ['pastel','husl','Set2','flare','crest','magma','icefire']\n",
+    "    ),\n",
+    "    hue = widgets.ToggleButtons(\n",
+    "        options = ['race','gender'],\n",
+    "        disabled = False,\n",
+    "        button_style = 'success'\n",
+    "    ),\n",
+    "    bins = widgets.IntSlider(\n",
+    "        value = 10,\n",
+    "        min = 3,\n",
+    "        max = 15,\n",
+    "        step = 1\n",
+    "    ),\n",
+    "    x_range_1 = widgets.IntRangeSlider(\n",
+    "        value = [1900,2000], \n",
+    "        min = 1900,\n",
+    "        max = 2000,\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
probably want to get rid of the last blank cell.

Also, would it make sense to add the "provenence" stuff at the end as we often do with our examples?